MIP/mip-103/README.md

```diff
@@ -0,0 +1,257 @@
+
+## Motivation
+
+Ultimately, the role of the contracts in the current FFS protocol are simply to verify and mark consensus on a particular commitment at a particular `superBlock` height, or round, on a given ledger. While implementing the actual tabulation of votes on-chain serves to reduce development time and avoid the implementation of an off-chain service, it does not in fact minimize the role of the on-chain contracts. Baker Confirmations instead propose such minimization by ensuring that the contracts simply verify ZK proofs of consensus and dispatch the appropriate onchain events.
+
+## Specification
+
+![Baker Confirmations](./rsc/baker-confirmations.png)
+
+There are two crucial roles in a Baker Confirmer which we will address individually and in detail:
+
+- **Attesters** generate commitments at a given block-height and sign them before broadcasting them to the Baker Network.
+- **Validators** verify the commitments generated by the Attesters, tabulate them in a Zero Knowledge program, and submit the proof of this Zero Knowledge program and its outputs on-chain.
+
+In a simplified model of a Baker Network, all Participants are both Attesters and Validators. However, as we will demonstrate, this is not a strict requirement of the protocol.
+
+For simplicity we specify Baker Confirmations in terms of a simple unidirectional commitment scheme we call [Baker Coin](#baker-coin). We introduce but do not specify generalizations of this scheme which are especially intriguing for their composability, modularity, and multi-directionality.
+
+Finally, we describe how the specification addresses problems [MD-3](https://github.com/movementlabsxyz/MIP/pull/3), [MD-4](https://github.com/movementlabsxyz/MIP/pull/4), and [MD-5](https://github.com/movementlabsxyz/MIP/pull/5), as well as contending with problems inherent to the Baker Confirmations scheme itself. 
+
+### Baker Coin
+
+Baker Coin is a class of unidirectional commitment protocols satisfying the [postconfirmations]() scheme proposed in [MIP-37](https://github.com/movementlabsxyz/MIP/pull/37). It has the following additional constraints:
+
+1. **Unidirectional**: A Baker Coin protocol MUST intend ONLY to post a commitment the state of a given Ledger B onto Ledger A, not vice versa. 
+2. **Staked**: A Baker Coin protocol MUST rely on weighted votes, known as stake, to determine whether consensus has been reached amongst a set of participants. These stake weights must be represented over the field of integers from 0 to 2^256 - 1, $\mathbb{F}_{2^{256}}$.
```

Comment:
Here and in the figure you refer to participants as actors in the Baker protocol (off-Ledger A).. However you also refer to A-participants as participants. Are these identical ?